### DISCUSSION TITLE
Could useNoInteractiveChildren be exported?

### DISCUSSION THREAD

**Discussion by janhassel** [carbon-design-system/carbon#13321] [Ideas]
I'm interested if you've considered exporting the `useNoInteractiveChildren` react hook as I see this being helpful especially for PAL teams to re-use the existing logic for their components as well.

**Comment by tay1orjones** [comment]
I'm not opposed but we generally don't export internals so they can change without having to flag a breaking change. 

Might be worth considering exporting these under their own new package, `@carbon/tools` maybe or something? `@carbon/hooks`? I'm not sure. It would be less of an issue to mark breaking changes in a separate package of helpers like that.

**Reply by janhassel** [reply]
Okay, that make sense! Let's leave it for now.
